src/app/shared/components/alert/alert-object.ts

```diff
@@ -20,6 +20,11 @@ export interface IAlertObjectParams {
    */
   type?: string;
 
+  /**
+   * size of Alert - if true, Alert is single line. If false, alert is multiple lines
+   */
+  smallAlert?: boolean;
```

Comment:
do we still use alerts or is it exclusively toast now? I don't remember seeing any in testing

Reply:
removing alerts, toasts indeed way better